releases/release-5.4.2.md

```diff
@@ -0,0 +1,118 @@
+
+    + PingCAP/TiCDC
+
+        (dup: release-6.1.0.md > Bug fixes> Tools> TiCDC)- Fix data loss that occurs in special incremental scanning scenarios [#5468](https://github.com/pingcap/tiflow/issues/5468)
+        - Fix a bug in redo log manager that flush log executed before writing logs [#5486](https://github.com/pingcap/tiflow/issues/5486)
+        - Fix a bug that resolved ts moves too fast when part of tables are not maintained redo writer. [#5486](https://github.com/pingcap/tiflow/issues/5486)
+        - Add uuid suffix to redo log file name to prevent name conflict, which may cause data loss. [#5486](https://github.com/pingcap/tiflow/issues/5486)
+        - Fix replication interruption due to leader missing by extending region retry duration
+        - Fix min resolved ts/checkpoint table ID metrics [#5542](https://github.com/pingcap/tiflow/pull/5542)
+        - `Fix a bug that mysql sink may save a wrong checkpointTs`. [#5107](https://github.com/pingcap/tiflow/issues/5107)
+        - Fix a bug that may causes goroutine leak in http server. [#5303](https://github.com/pingcap/tiflow/issues/5303)
+        - `None` [#2792](https://github.com/pingcap/tiflow/issues/2792)
+
+    + TiDB Data Migration (DM)
+
+        (dup: release-6.1.0.md > Bug fixes> Tools> TiDB Data Migration (DM))- Fix the issue that DM occupies more disk space after the task automatically resumes [#3734](https://github.com/pingcap/tiflow/issues/3734) [#5344](https://github.com/pingcap/tiflow/issues/5344)
+        (dup: release-6.1.0.md > Bug fixes> Tools> TiDB Data Migration (DM))- Fix an issue that the uppercase table cannot be replicated when `case-sensitive: true` is not set [#5255](https://github.com/pingcap/tiflow/issues/5255)
+        - `None`. [#4858](https://github.com/pingcap/tiflow/issues/4858)
+
+## Compatibility change(s)
+
+## Improvements
+
++ TiDB
+
+    <!--transaction-->
+    - Improve the recovery time if the tikv hibernate region is enabled [#34906](https://github.com/pingcap/tidb/issues/34906)
+
++ TiKV
+
+    - Reload TLS certificate automatically when it changes. [#12546](https://github.com/tikv/tikv/issues/12546)
+    (dup: release-6.1.0.md > Improvements> TiKV)- Improve the health check to detect unavailable Raftstore, so that the TiKV client can update Region Cache in time [#12398](https://github.com/tikv/tikv/issues/12398)
+    - Use `posix_fallocate` for space reservation. [#12543](https://github.com/tikv/tikv/issues/12543)
+    (dup: release-5.2.4.md > Improvements> TiKV)- Transfer the leadership to CDC observer to reduce latency jitter [#12111](https://github.com/tikv/tikv/issues/12111)
+
++ PD
+
+    (dup: release-6.1.0.md > Improvements> PD)- Disable compiling swagger server by default [#4932](https://github.com/tikv/pd/issues/4932)
+
+## Bug Fixes
+
++ TiDB
+
```

Comment:
<!--sql-infra-->
    - Fix TiDB does not free the resources of the disconnected session correctly when a network connectivity event occurred. [#34722](https://github.com/pingcap/tidb/issues/34722)
    - Fix querying CTE views unexpectedly reports `references invalid table`. [#33965](https://github.com/pingcap/tidb/issues/33965)